roles/jenkins_builder/tasks/pkgs.yml

```diff
@@ -8,6 +8,7 @@
   - dbench
   - yajl
   - sudo
+  - gcc  
```

Comment:
Well, I suspect gcc is installed by chance. On the *bsd, that's part of the base system, on Linux, that's only because we pull rpm-build. Wouldn't it be cleaner to have it excplitely installed ?